lib/features/mailbox_dashboard/presentation/controller/search_controller.dart

```diff
@@ -51,6 +51,7 @@ class SearchController extends BaseController with DateRangePickerMixin {
   SearchQuery? get searchQuery => searchEmailFilter.value.text;
 
   FocusNode searchFocus = FocusNode();
+  String currentSearchText = '';
```

Comment:
why do you need it?

Reply:
To highlight the search suggestion in web view, according to the search query trigger by debouncer.